package.json

```
@@ -85,6 +85,7 @@
         "set-value": ">=4.0.1",
         "tar": ">=6.1.9",
         "jszip": ">=3.7.0",
+        "shelljs": "^0.8.5",
```

It would be nice to use a more specific key for this, since there's multiple transitive dependencies on shelljs and only one of them is so far behind as to require a resolution.
```suggestion
        "**/azure-pipelines-task-lib/shelljs": "^0.8.5",
```